src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -144,7 +185,7 @@ describe('workspace service', () => {
           page: 1,
         })
         .expect(200);
-      expect(listResult.body.result.total).toEqual(3);
+      expect(listResult.body.result.total).toEqual(1);
```

Comment:
For list, here the total should be 3 as we will create reserved workspace in the list call.

Reply:
since we will clear the data after each test, so the result will be 1. the number 3 could pass previous check because we have permission check enabled, data is not clean up after each run.